src/c-writer.cc

```diff
@@ -428,6 +428,7 @@ static constexpr char kParamSuffix =
 static constexpr char kLabelSuffix = kParamSuffix + 1;
 
 static constexpr char kSymbolPrefix[] = "w2c_";
+static constexpr char kLocalSymbolPrefix[] = "var_";
```

Comment:
I guess unlike the other prefixes, this one is 100% internal so changing it is not a user-visible change?

Reply:
Yes. So should I change `kLocalSymbolPrefix` to be a private field of `CWriter`?

Reply:
So sorry, I just meant that if we decided to change `var_` to `local_` in the figure it would not be user visible?

This change lgtm.